packages/react/select/src/Select.tsx

```diff
@@ -196,8 +201,9 @@ interface SelectTriggerProps extends PrimitiveButtonProps {}
 
 const SelectTrigger = React.forwardRef<SelectTriggerElement, SelectTriggerProps>(
   (props: ScopedProps<SelectTriggerProps>, forwardedRef) => {
-    const { __scopeSelect, disabled = false, ...triggerProps } = props;
+    const { __scopeSelect, disabled: disabledProp = false, ...triggerProps } = props;
     const context = useSelectContext(TRIGGER_NAME, __scopeSelect);
+    const disabled = disabledProp || context.disabled;
```

Comment:
Sorry my bad, just realised we do it slightly differently in `RadioGroup` so let's keep it consistent (we want context to take over):

```suggestion
    const { __scopeSelect, disabled = false, ...triggerProps } = props;
    const context = useSelectContext(TRIGGER_NAME, __scopeSelect);
    const disabled = context.disabled || disabled;
```

Reply:
I'm not sure if it's possible to redeclare the `disabled`. So I look up the `RadioGroup`, and it's using `isDisabled` instead of `disabled`. Should I change it to `isDisabled` as well?

Reply:
Nvm, I did it anyway 😅

Reply:
Yes sorry that's what I meant to write 😀